sqlserver/datadog_checks/sqlserver/sqlserver.py

```diff
@@ -165,7 +165,20 @@ def __init__(self, name, init_config, instances):
         self.instance_metrics = []
         self.instance_per_type_metrics = defaultdict(list)
         self.do_check = True
+
+        self.proc = self.instance.get('stored_procedure')
         self.proc_type_mapping = {'gauge': self.gauge, 'rate': self.rate, 'histogram': self.histogram}
+        self.custom_queries = []
+        custom_queries = self.instance.get('custom_queries')
+
+        if self.proc and custom_queries:
+            msg = 'Stored Procedures and Custom Queries cannot be part of the same check instance.'
```

Comment:
Any reason?

Reply:
Initially because there were logical differences to how things got collected.  Given the resulting implementation though, they could be combined easily, will post update.